File: Python/busquedalineal.py
inputArr= [11,21,31,41,51,61]
print("El array es ")
print("--- Recorrido secuencial del array ---")
for j in range(len(inputArr)):
    print(inputArr[j], end=" ")
    
numbuscar=int(input("Que numero quiere buscar?"))
for j in range(len(inputArr)):
    if inputArr[j]==numbuscar:
        print("El numero esta en el lugar:",j+1)
        break
else:
    print("El numero no esta en el array")

File: c++/busquedalineal.cpp
#include <iostream>
using namespace std;
int main(){
    int array[6]={11,21,31,41,51,61};
    int num=sizeof(array)/sizeof(array[0]);
    int numbuscar;

    cout<<"---Busqueda secuencial---\n";
    cout<<"El array es: ";
    for(int i=0;i<num;i++)
    {
        cout<<array[i]<<" ";
    }

    cout<<"\nEscribe el numero a buscar: ";
    cin>>numbuscar;

    int i;
    for(i=0; i<num; i++){
        if(array[i]==numbuscar){
            cout<<"El numero esta en el lugar: "<<i+1<<endl;
            break;
        }
    }

    if(i==num){
        cout<<"El numero no esta en el array"<<endl;
    }

    return 0;
}